lib/workers/repository/onboarding/pr/errors-warnings.ts

```diff
@@ -4,12 +4,13 @@ import type { PackageFile } from '../../../../modules/manager/types';
 import { emojify } from '../../../../util/emoji';
 
 export function getWarnings(config: RenovateConfig): string {
-  if (!config.warnings.length) {
```

Comment:
why not simply `config.warnings?.length` ?

typescript 4.7 should smart enough to handle it